hypha/apply/funds/models/submissions.py

```diff
@@ -770,13 +774,21 @@ def log_status_update(sender, **kwargs):
     notify = kwargs['method_kwargs'].get('notify', True)
 
     if request and notify:
-        messenger(
-            MESSAGES.TRANSITION,
-            user=by,
-            request=request,
-            source=instance,
-            related=old_phase,
-        )
+        if kwargs['source'] == DRAFT_STATE:
+            messenger(
+                MESSAGES.NEW_SUBMISSION,
+                request=request,
+                user=by,
+                source=instance,
+            )
```

Comment:
Why do we add the notification here but disable it below in "render_landing_page"? Want to understand the code.

Reply:
Currently, in production, we notify OTF slack for a new submission, other messages using NEW_SUBMISSION. But now if the user saves the application as a draft then we do not send NEW_SUBMISSION message. Therefore disabled in "render_landing_page". But it will be sent later when the user edits the draft application and send for review.

Reply:
Now I understand, thanks!